File: android/app/src/main/java/br/dev/marconi/lyricalia/repositories/match/PlayerPodium.kt
package br.dev.marconi.lyricalia.repositories.match


import android.os.Parcelable
import kotlinx.parcelize.Parcelize
import kotlinx.serialization.Serializable

@Serializable
data class PlayerPodium (
    val id: String,
    val score: Int,
    val submission: List<String>
)

File: api/Sources/api/Models/player_podium.swift
import Vapor

final class PlayerPodium: Codable {
    let id: UUID
    let score: Int
    let submission: [String]?

    init(id: UUID, score: Int, submission: [String]? = nil) {
        self.id = id
        self.score = score
        self.submission = submission
    }
}
